src/components/pages/AddItemPage/index.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export const Add = () => <h1>Add Item</h1>;
```

Comment:
Good movement! Pages are more independent and isolated! Good benefits for the future 🚀 !